exts/virtio_blk/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "virtio_blk"
+version = "0.0.1"
+authors = ["The Kerla Authors"]
```

Comment:
As it's mostly your authorship at the moment, I think it's perfectly legal to claim your name here. There is no legal entity behind Kerla at the moment other than individual contributors and definitely one under such name don't exist.

Reply:
I agree that it is pretty ugly. I was just adhering to conformance with the Virtio spec and Qemu. I don't mind making it more legible.